src/index.html

```diff
@@ -3,11 +3,286 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Bose</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="icon" type="image/x-icon" href="/images/favicon.ico">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;700;800&display=swap" rel="stylesheet">
     <script src="scripts/main.js" defer></script>
   </head>
   <body>
-    <h1>Miami</h1>
+    <h1 hidden>Bose</h1>
+    <header class="header">
+      <div class="header__wrapper">
+        <div class="header__menu">
+          <a href="#" class="header__logo" >
+            <img
+              src="./images/logo.svg"
+              alt="BOSE logo"
+              class="header__logo-img"
+            >
+          </a>
+          <div class="header__navigation">
+            <a href="#" class="header__phone"></a>
+            <button class="header__menu-button header__burger"></button>
+            <button class="header__menu-button header__cross"></button>
+          </div>
+        </div>
+        <nav class="nav header__nav">
```

Comment:
when menu is on - it should cover all viewport and page should not scroll
![menu cover no scroll](https://user-images.githubusercontent.com/39308717/162926513-1f942b1e-2adf-48c8-9aec-07b93566b7b5.png)